eks-anywhere-common/Addons/Partner/SUSE/NeuVector/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: neuvector
```

Comment:
Annotation mention here in this link https://github.com/aws-samples/eks-anywhere-addons/blob/main/README.md#:~:text=apiVersion%3A%20v1%0Akind%3A%20Namespace%0Ametadata%3A%0A%20%20%20%20name%3A%20kubecost%0A%20%20%20%20labels%3A%0A%20%20%20%20%20%20%20%20aws.conformance.vendor%3A%20kubecost%0A%20%20%20%20%20%20%20%20aws.conformance.vendor%2Dsolution%3A%20cost%2Danalyzer as mentioned in the readme link is missing in the PR